src/test/java/Homework21.java

```diff
@@ -0,0 +1,40 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.annotations.Test;
+
+public class Homework21 extends BaseTest {
+String newPlaylistName = "PlayMusic";
+    @Test
+    public void renamePlaylist() {
+        provideEmail("elliott.nance@testpro.io");
+        providePassword("te$t$tudent");
+        submit();
+        contextClickChosenPlaylist();
+        clickEditBtn();
+        provideNewPlaylistName();
+        doesPlaylistExist();
+        //Assert.assertTrue(doesPlaylistExist());
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**